tests/test_models.py

```diff
@@ -332,3 +332,24 @@ def test_contrast_bad_ratio():
         ),
     ):
         RATapi.models.Contrast(name="My Contrast", domain_ratio="bad ratio")
+
+
+@pytest.mark.skip("Type change is not currently supported")
+@pytest.mark.parametrize("model", [RATapi.models.Background, RATapi.models.Resolution])
```

Comment:
This needs updating in light of the changes to freezing the fields instead of wiping values.